Marlin/src/module/configuration_store.cpp

```diff
@@ -154,7 +154,9 @@ typedef struct SettingsDataStruct {
   // HAS_BED_PROBE
   //
   float zprobe_zoffset;                                 // M851 Z
-
+  #if ENABLED(INDEPENDENT_Z_OFFSETS) && ENABLED(DUAL_X_CARRIAGE)
+    float x2_zprobe_zoffset;
+  #endif
```

Comment:
If there is to be either `zprobe_zoffset` or `x1/x2_zprobe_zoffset` but not both, that should also be reflected in the structure here.

This `#endif` should be followed by a blank line.

Reply:
Updated, originally just looked at it as ensuring size matched, and the first 2 are used interchangeably in the save / restore scenario.